lib/tasks/insights.rake

```diff
@@ -24,13 +24,19 @@ namespace :rh_cloud_insights do
   desc "Re-announce all organizations into Sources on RH cloud."
   task announce_to_sources: [:environment] do
     logger = Logging::Logger.new(STDOUT)
+    local_advisor = ForemanRhCloud.with_local_advisor_engine?
     Organization.unscoped.each do |org|
+      next if local_advisor
```

Comment:
Is it better to add this line before `Organization.unscoped.each` we don't need to do the each right

Reply:
Enclosed in a conditional instead.